src/test/resources/Homework18.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+<suite name="Homework18">
+    <parameter name="BaseURL" value="https;//qa.koel.app./">
+
+    </parameter>
+
+    <test name="Example Test" preserve-order="false">
+        <classes>
+            <class name="LoginTests"/>
+            <class name="BaseTest"/>
+            <class name="Homework18">
```

Comment:
Hi,
I have checked your homework 18, Homework18 test script passed but in this Homework #18:
XML File Configurations required Specify the @Test method to be ran in the new XML file. and  Use the <methods> & <include> tags. But I can't see you add `<methods> & <include>` to `Homework18.xml` file.

TestNG provides the most useful functions for users when fully flexible in the config to optionally add or remove tests. 
//HINT:  Method to be run must use <include>
```
<class name="Homework18">
                <methods>
                    <include name="playSong"/>
                </methods>
            </class>
```